homeassistant/components/homematic/entity.py

```diff
@@ -115,10 +116,21 @@ def _hm_event_callback(self, device, caller, attribute, value):
 
         # Is data needed for this instance?
         if attribute in self._data:
-            # Did data change?
-            if self._data[attribute] != value:
-                self._data[attribute] = value
-                has_changed = True
+            if (
+                attribute in self._channel_map
+                and device == self._channel_map[attribute]
+            ):
+                # Did data change?
+                if self._data[attribute] != value:
+                    self._data[attribute] = value
+                    has_changed = True
+            else:
```

Comment:
I don't think that make sense to log that with debug if we not able to fix it

Reply:
Yes, I have just copied and testes the code that was presented at the mentioned issue. We don't necessarily need that info.